sdk/keyvault/ci.yml

```diff
@@ -41,6 +41,9 @@ stages:
         - Name: azure-security-keyvault-certificates
           Path: azure-security-keyvault-certificates
           VcpkgPortName: azure-security-keyvault-certificates-cpp
+        - Name: azure-security-keyvault-administration
+          Path: azure-security-keyvault-administration
+          VcpkgPortName: azure-security-keyvault-administration-cpp
```

Comment:
I am not as familiar with keyvault CI yaml. Are these two the only changes needed to light up CI runs?

Are nightly/internal pipelines running?

Reply:
tey are running , it's about publishing artifacts in the piplines.